scripts/zones/Jugner_Forest/MobIDs.lua

```diff
@@ -1 +1,12 @@
 FRAELISSA               = 17203447;
+KING_ARTHRO             = 17203216;
+KNIGHT_CRAB1            = 17203206;
```

Comment:
no need for individual ID's for all 10, we can offset from arthro's ID: example 10th crab is arrthro-11st crab is arthro -10

Reply:
I like this idea. Thanks for mentioning it, will work to do this now.